Website/DesktopModules/Hotcakes/Core/Controllers/CheckoutController.PaymentError.cs

```diff
@@ -44,6 +45,17 @@ public partial class CheckoutController
 		public ActionResult PaymentError()
 		{
 			CheckoutViewModel model = PaymentErrorSetup();
+            if (HccApp.CurrentStore.Settings.PaymentCreditCardGateway == "15011DF5-13DA-42BE-9DFF-31C71ED64D4A")
+            {
```

Comment:
There are a lot of values here without any defensive coding.  Think about your testing...  Do you think that's okay?  If you think yes, that's fine.  It's just a question.  :)

Reply:
This info is needed by stripe to work on the client side. There is no problem with exposing this data